packages/create-plugin/scripts/generate-app.js

```diff
@@ -0,0 +1,24 @@
+const nodePlop = require('node-plop');
```

Comment:
Note that I decided to copy-paste the scripts rather than trying to abstract common parts into a separate function to keep them simple and clear.

One could think of a single-script that handle CLI arguments but that will require a far more complex logic that we will later have to maintain and understand so I decided against.